climpred/metrics.py

```diff
@@ -394,12 +562,14 @@ def _nmae(forecast, reference, dim='svd', comparison=None):
     mae_skill = _mae(forecast, reference, dim=dim)
     # TODO: check if this is the expected normalization
```

Comment:
Make sure to do this before we merge, since it seems important to having this be an accurate metric.

Reply:
checked and corrected